ConcernsCaseWork/ConcernsCaseWork.CypressTests/package.json

```diff
@@ -19,7 +19,7 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "cypress": "^9.7.0",
```

Comment:
If I'm reading this change properly we have downgraded cypress ?

Reply:
I had an issue which meant that I needed to downgrade Cypress, I have just fixed the issue and its back to Cypress 9.7.0 now :), potentially we should be on V10 but will upgrade at a later stage hopefully.